client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+
+export const SeoSiteSearchPreview = React.createClass( {
+	shouldComponentUpdate( next ) {
+		const current = this.props;
+
+		return ! (
+			current.snippet === next.snippet &&
+			current.title === next.title &&
+			current.url === next.url
+		);
+	},
+
+	render() {
+		const {
+			snippet,
+			title,
+			url
+		} = this.props;
+
+		return (
+			<div className="seo-search-result-preview">
```

Comment:
Since the component fragment needs to match the folder name, you could either nest a folder `seo-search-result-preview` or rename the current folder and keep the existing names.